Common.pm

```diff
@@ -204,6 +204,9 @@ sub get_primitive_base($;$) {
     my ($tag, $default) = @_;
 
     my $base = $tag->getAttribute('base-type') || $default || 'uint32_t';
+    if ($base =~ /u?int[136]?[2468]_t/) {
```

Comment:
This seems like it would apply to most headers - aren't there already commonly-included headers before any DF headers where this could go?